docs/installing-northstar/northstar-installers/vtol-guide.md

```diff
@@ -72,27 +83,34 @@ You can also right click on a specific mod to get the option to either delete it
 
 ![Mods browser](../../images/vtol-mods-browser.png)
 
-VTOL has a built in-browser for searching [Northstar's Thunderstore page](https://northstar.thunderstore.io/) (the website where most Northstar mods get uploaded to), located at the second tab _"Browse"_ on the tab list. You will see the newest uploaded mods here by default, which you can change by applying filters (Note: The sorting is by default Low-High for filters. You can change this by pressing the button with an arrow next to the filters button, making it High-Low)
+VTOL has a built in-browser for searching [Northstar's Thunderstore page](https://northstar.thunderstore.io/) (the website where most Northstar mods get uploaded to), located at the second tab _"Browse"_ on the tab list. 
+You will see the newest uploaded mods here by default, which you can change by applying filters (Note: The sorting is by default Low-High for filters. 
+You can change this by pressing the button with an arrow next to the filters button, making it High-Low)
 
 VTOL also has support where all you need to do is drag and drop a mod downloaded from the Thunderstore page onto the _"Mods"_ tab (the second tab on the left side) to automatically install the mod.
 
 ## Settings
 
 ![VTOL Settings](../../images/vtol-settings-tab.png)
 
-On the way bottom of the tab list, there is a button to open settings for the VTOL manager. You will want to keep most of these on default, but change them as you wish. It is normal for the _"Restart_As_Admin"_ button to appear untoggled. This is also where you can tick _"Enable_EA_APP_Usage"_ to make the client work better with the EA app, assuming you have it and not Origin.
+On the way bottom of the tab list, there is a button to open settings for the VTOL manager. You will want to keep most of these on default, but change them as you wish.
+It is normal for the _"Restart_As_Admin"_ button to appear untoggled. This is also where you can tick _"Enable_EA_APP_Usage"_ to make the client work better with the EA app, assuming you have it and not Origin.
 
 ## Skins
 
 ![Skins installation tab](../../images/vtol-skins.png)
 
-The third settings tab on VTOL is the _"Skins"_ tab. VTOL has support for skins formatted for an older version of Northstar, normally requiring an extra tool. All you need to do is drag the folder of one of these old style skins onto the Skins screen to install it automatically.
+The third settings tab on VTOL is the _"Skins"_ tab. VTOL has support for DDS format skins (the skins with the numbered folders in them), which are commonly used for Vanilla Titanfall 2, with Northstar generally opting to convert them to mods.
+
+This format is older, and less consistent than converting them to a mod to be used with Northstar (see [Advocate](./#advocate)), and can still be used on Vanilla by making use of a [vanilla profile](../using-northstar/advanced.md#vanilla-profile).
```

Comment:
```suggestion
This format is older and less consistent than converting them to a mod to be used with Northstar (see [Advocate](./#advocate)). It can still be used on Vanilla by making use of a [vanilla profile](../using-northstar/advanced.md#vanilla-profile).
```